OrbitGl/CaptureSerializer.cpp

```diff
@@ -170,13 +170,13 @@ void FillEventBuffer() {
 }
 
 void CaptureSerializer::ProcessCaptureData(const CaptureInfo& capture_info) {
-  // Functions
-  Capture::GSelectedInCaptureFunctions.clear();
+  // Clear the old capture
+  Capture::capture_data_ = CaptureData();
```

Comment:
this is a bit confusing. maybe have a local initialize it and then at the end of this function assign to global? That way you do not need to always mention Capture::capture_data_ below.

Reply:
Changed the code completely. Should be fine now.